### PR TITLE
fix(sinon-chai): Added once commands for sinon methods

### PR DESCRIPTION
**Pull request by sainthkh**
Related to cypress-io/cypress#9644

### CODE REVIEW

File: lib/sinon-chai.js [hunk 1]
```diff
@@ -181,7 +181,9 @@
     sinonMethod("calledAfter", "been called after %1");
     sinonMethod("calledOn", "been called with %1 as this", "", ", but it was called with %t instead");
     sinonMethod("calledWith", "been called with arguments %*");
+    sinonMethod("calledOnceWith", "been called exactly once with arguments %*");
```
Comment (Andarist) [inline lib/sinon-chai.js:184]: I was wondering - shouldnt this report an error based on a failed criteria? The user wont easily know now if the function has been called more than once or if it has been called with wrong arguments

Comment (sainthkh) [reply lib/sinon-chai.js:184]: I manually tested them. Users can know if a function is called more than once or with wrong arguments.

* Called more than once:

Code: 
```js
it('t', () => {
  const foo = cy.spy().as('foo')

  setTimeout(() => {
    foo({ bar: 'test' })
    foo({ bar: 'test' })
  }, 500)

  cy.get('@foo').should(
    'be.calledOnceWith',
    Cypress.sinon.match({
      bar: 'test',
    }),
  )
})
```

Result:
Screenshot from 2020-12-15 16-02-20



* Wrong arguments:

Code: 
```js
it('t', () => {
  const foo = cy.spy().as('foo')

  setTimeout(() => {
    foo({ bar: 'testing' })
  }, 500)

  cy.get('@foo').should(
    'be.calledOnceWith',
    Cypress.sinon.match({
      bar: 'test',
    }),
  )
})
```

Result:
Screenshot from 2020-12-15 16-03-58

Comment (Andarist) [reply lib/sinon-chai.js:184]: Ok, this addresses my concerns - although I believe that the first part of the error message could be more clear about what has failed. One has to deduce it from the "following calls were made" part. Having the `Cypress.sinon.match` object stringified in a way it is stringified doesn't help much either.

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**